exercises/etl/metadata.toml

```diff
@@ -1,4 +1,4 @@
-title = "ETL"
-blurb = "We are going to do the `Transform` step of an Extract-Transform-Load."
-source = "Exercise by the JumpstartLab team for students at The Turing School of Software and Design."
+title = "Lexiconia"
```

Comment:
Jeremy just announced ETL is part of this month's 12in23.

Should the title match the path? Should we be changing the title this month?

Reply:
No, we definitely shouldn't.

Reply:
```suggestion
title = "ETL"
```